backend/jottacloud/jottacloud.go

```diff
@@ -972,6 +1001,9 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 		CanHaveEmptyDirectories: true,
 		ReadMimeType:            true,
 		WriteMimeType:           false,
+		ReadMetadata:            true,
+		WriteMetadata:           true,
+		UserMetadata:            false,
```

Comment:
> I think this is missing the metadata flags in Fs.Features()?

@ncw did you miss these, or did I miss something? 😄

Reply:
I missed them sorry - I blame my old, tired eyeballs ;-)